cypress/pageobject/Users/UserCreation.ts

```diff
@@ -59,12 +74,12 @@ export class UserCreationPage {
     inputElementId: string,
     dateValue: string,
   ) {
-    this.clickElementById(dateElementId);
+    cy.get("#" + dateElementId).click();
```

Comment:
we are reusable command.ts function, refer it

Reply:
It is just a click. I dont think we have function for simple click in data selection